src/org/montclairrobotics/sprocket/pipeline/Pipeline.java

```diff
@@ -0,0 +1,34 @@
+package org.montclairrobotics.sprocket.pipeline;
+
+import java.util.ArrayList;
+
+public class Pipeline <T>{
```

Comment:
You could make it implement Updatable but not add it to the Updater. That way we have some sense of consistency but we still maintain control over when we run the pipeline.